notebooks/vertex_genai/solutions/vertex_llm_evaluation.ipynb

```diff
@@ -0,0 +1,1055 @@
+{
```

Comment:
<p><code>{task}@{version}</code> -&gt; <code>{task}</code> ?</p><p> </p><p>Adding the expected parameters under <code>autorater_prompt_parameters</code> would be helpful.</p><p> </p><p>Also, I'd add that we can simply specify models (<code>model_a</code> and <code>model_b</code> ) instead of specifying pre-generated responses (<code>response_column_a</code> and <code>response_column_b</code>). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/443/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Done!</p>